en/developer_guide/multiPanel.rst

```diff
@@ -0,0 +1,75 @@
+Change an administration view to multi panel
+============================================
+
+Context
+-------
+
+When working with administration choice, sometimes you need to add your own forms or contents to the existing one.
+Open Orchestra implement a panel system to make this work easier.
+Here is an example of how we use this panel system to add a workflow panel to Users page.
```

Comment:
I don't understand what this feature will bring. Is it about the display, the code, or both ?